src/objects/js/components/admin/permissions/permission-form.js

```diff
@@ -35,6 +69,7 @@ const PermissionForm = ({objectFields, dataFieldChoices, tokenChoices, objecttyp
                 errors={errors["object_type"]}
                 onChange={(value) => {
                     setObjectType(value);
+                    fetchObjecttypeVersions(value);
```

Comment:
Is this needed? Currently two requests are made when the objecttype is changed, I think the `useEffect` hook above is probably sufficient.